docs/data-model.asciidoc

```diff
@@ -354,50 +452,118 @@ These metric documents can be identified by searching for `metricset.name: servi
+
+The `@timestamp` field of these documents holds the start of the aggregation interval.
 
 [float]
-==== Example metric document
+===== Example service-summary document
 
-This example shows what metric documents can look like when indexed in {es}.
+This example shows what service-summary documents can look like when indexed in {es}.
 
 [%collapsible]
 .Expand {es} document
 ====
 
-This example contains JVM metrics produced by the {apm-java-agent}.
-and contains two related metrics: `jvm.gc.time` and `jvm.gc.count`. These are accompanied by various fields describing
-the environment in which the metrics were captured: service name, host name, Kubernetes pod UID, container ID, process ID, and more.
-These fields make it possible to search and aggregate across various dimensions, such as by service, host, and Kubernetes pod.
-
 [source,json]
 ----
-include::./data/elasticsearch/metricset.json[]
+include::./data/elasticsearch/service_summary_metric.json[]
 ----
 ====
 
+[float]
+==== Data streams
+
+Metrics are stored in the following data streams:
+
+include::./data-streams.asciidoc[tag=metrics-data-streams]
+
+See <<apm-data-streams>> to learn more.
+
+[float]
+==== Aggregated metrics: limits and overflows
+
+For all aggregated metrics, namely transaction, service-transaction, service-destination, and service-summary metrics,
+there are limits on the number of unique groups tracked at any given time.
+
+[float]
+===== Limits
+
+Note that all the below limits may change in the future with further improvements.
+
+* For transaction metrics, the limits are 1000 services per GB of APM Server, and 5000 transaction
+groups per GB of APM Server. Additionally, each service may only consume up to 10% of the transaction groups,
+which is 500 transaction groups per service per GB of APM Server.
+* For service-transaction metrics, the limit is 1000 service transaction groups per GB of APM Server.
+* For service-destination metrics, the limit is a constant of 10000 service destination groups.
+* For service-summary metrics, the limit is 1000 service summary groups per GB of APM Server.
+
+[float]
+===== Overflows
+
+When a dimension has a high cardinality and exceeds the limit, the metrics will be aggregated
+under a dedicated overflow bucket.
+
+For example, when `transaction.name` has a lot of unique values and reaches the limit
+of unique transaction groups tracked, any transactions with new `transaction.name` will be aggregated under
+`transaction.name`: `_other`.
+
+Another example of how the transaction group limit may be reached is if `transaction.name` contains just a few unique values,
+but the service is deployed on a lot of different hosts. As `host.name` is part of the aggregation key for transaction metrics,
+the max transaction group limit is reached for a service that has 100 instances and 40 different transaction names when
+connected to an APM Server with 8GB of RAM. Once this limit is reached, any new combinations of `transaction.name` and
```

Comment:
That example doesn't account for the transaction result also being dimension?

```suggestion
the max transaction group limit is reached for a service that has 100 instances, 10 different transaction names, and 4 transaction results per transaction name when
connected to an APM Server with 8GB of RAM. Once this limit is reached, any new combinations of `transaction.name` and
```

cc @dgieselaar

Reply:
hmm, do we want to update the next sentence as well to avoid confusion?
>  any new combinations of `transaction.name` and...

Reply:
took the liberty to update the following sentence to include `transaction.result` as well.

Reply:
thanks!